### ISSUE TITLE
ZXing 3.3.2 snapshot still creates wrong barcode for text with leading asterisk

### ISSUE THREAD

**Issue by CharlyNapf** [zxing/zxing#944]
Hi,

this is a follow-up of issue #806 and #923. Now I am testing a not official snapshot of version 3.3.2 and it does NOT solve the error reported in #923.
My response from 18-Dec-2018 was wrong, sorry.
See ![inf_11_testedwithzxing332](https://user-images.githubusercontent.com/23717092/35038320-c3605de2-fb7a-11e7-90fb-fd67e22114c9.png)

**Comment by CharlyNapf** [comment]
Does anybody care for this issue? Will it be fixed in release 3.3.2?

**Comment by srowen** [comment]
Nobody works on the project but I can look at pull requests. It is DIY here.

**Comment by oehhar** [comment]
Maybe, you may consider to look to this version of zint:

[sourceforge zint](https://sourceforge.net/projects/zint/)

Best regards,
Harald

**Comment by axxel** [comment]
I tried your string "*CH/GN1/022/00" with the [patched c++ port](https://github.com/nu-book/zxing-cpp/pull/25/commits/ec2e5c6feb81c7b5d26be59d65a6ff43394cffd7) I mentioned earlier and it works as it should. I reverted that one-line-patch and it gave me exactly your wrong result "*CH/GN1/022/{00".

My conclusion: this issue is fixed in the code base (in both the c++-port and this Java project). If you still see it with your test-app then I guess something went wrong with the snapshot that @srowen provided or with your own build setup using that snapshot. Hinting at the second would be the fact that you first said the issue is fixed and later noticed it was not ;).
